platform/windows/windows_terminal_logger.cpp

```diff
@@ -54,6 +54,7 @@ void WindowsTerminalLogger::logv(const char *p_format, va_list p_list, bool p_er
 		return;
 
 	wchar_t *wbuf = (wchar_t *)malloc((len + 1) * sizeof(wchar_t));
```

Comment:
You forgot to change this one.

Reply:
When I change this to Godot's macro, the unit tests for Windows fail. Maybe it has to do with padding being enforced on debug builds?
![image](https://user-images.githubusercontent.com/6501975/110822735-45510480-8289-11eb-932d-d7da6b47375b.png)

Reply:
You can see if it works when using `memfree()` to free the memory too.